wolfssl/ssl.h

```diff
@@ -2080,9 +2107,6 @@ enum {
     BIO_NOCLOSE = 0,
 
     X509_FILETYPE_PEM = 8,
-    X509_LU_NONE      = WOLFSSL_X509_LU_NONE,
```

Comment:
If you end up doing what Sean suggests [here](https://github.com/wolfSSL/wolfssl/pull/4350#discussion_r698085460), could do the same here, so that we get both the enum and define.

Reply:
WOLFSSL_X509_LU_NONE is used as an enum in the code base now